web/src/features/fireWatch/fireWatchApi.ts

```diff
@@ -154,7 +61,7 @@ export interface FireWatchInput {
 export interface FireWatchOutput extends FireWatchInput {
   id: number
   create_timestamp: number
```

Comment:
We use iso strings for over the wire timestamps usually, any reason why we use a number over string?

Reply:
Force of habit from a past life, epoch times are hard to misinterpret. I think I introduced epoch time into our api somewhere else. I see what you mean about ISO strings being the standard for our API though, I'll switch things over.

Reply:
Fair, happy to revisit the precedent if it makes sense though.

Reply:
I made the change in https://github.com/bcgov/wps/pull/4493/commits/06cdb51a472c71884e1145918861c3f7efef1232.
I can't test in dev at the moment as WF1 test seems to be down.